src/views/LimitOrders/hooks/useGelatoLimitOrdersHistory.ts

```diff
@@ -52,13 +55,16 @@ const useOpenOrders = (turnOn: boolean): Order[] => {
     startFetch ? ['gelato', 'openOrders'] : null,
     async () => {
       try {
-        const orders = await gelatoLimitOrders.getOpenOrders(account.toLowerCase(), false)
+        const [openOrders, exeOrders] = await Promise.all([
+          gelatoLimitOrders.getCancelledOrders(account.toLowerCase(), false),
+          gelatoLimitOrders.getExecutedOrders(account.toLowerCase(), false),
+        ])
 
         syncOrderToLocalStorage({
-          orders,
+          orders: [...openOrders, ...exeOrders],
```

Comment:
we should both fetch open and executed orders to determine order status switched to executed